.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Auto-Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_id:
+        description: 'Commit ID'
+        required: true
+        default: 'HEAD'
+      version_number:
+        description: 'Version Number (Ex. 202000.00)'
+        required: true
+        default: '202000.00'
```

Comment:
Same here, having a default seems like it could trip things up, may as well force input

Reply:
Absolutely